pages/layouts/AppLayout.js

```diff
@@ -0,0 +1,16 @@
+import { Grid } from '@mui/material'
+import React from 'react'
+import Header from '../../templates/Header/Header'
+
+export const AppLayout = ({ children }) => {
+    return (
+        <>
```

Comment:
There is no need to use fragment tags here, you can just wrap whatever you'll render in this component in a Grid component